[Preview 8] [READY] Blazor HTML <head> tag control (#19276)
dotnet/AspNetCore.Docs: aspnetcore/blazor/fundamentals/additional-scenarios.md

The placeholder `{ELEMENT ID}` is the ID of the HTML element to display.

::: moniker range=">= aspnetcore-5.0"

## Influence HTML `<head>` tag elements

*This section applies to Blazor WebAssembly and Blazor Server.*

When rendered, the `Title`, `Link`, and `Meta` components add or update data in the HTML `<head>` tag elements:

```razor

> @using Microsoft.AspNetCore.Components.Web.Extensions

Reviewer: This was just updated to `Microsoft.AspNetCore.Components.Web.Extensions.Head` (https://github.com/dotnet/aspnetcore/pull/23833 is now merged, so further changes like this won't occur 😃).

```suggestion
@using Microsoft.AspNetCore.Components.Web.Extensions.Head
```

<Title Value="{TITLE}" />
<Link href="{URL}" rel="stylesheet" />
<Meta content="{DESCRIPTION}" name="description" />
```

> In the preceding example, placeholders for `{TITLE}`, `{URL}`, and `{DESCRIPTION}` are sting values, Razor variables, or Razor expressions.

Reviewer 2: `sting` -> `string` on line 237

(not sure I can directly edit this)

Author: I'll fix it. You can't edit directly ... it's a bit of a pain, but you have to branch from the PR's branch and then submit a PR back to it. It's faster 🏃 if I just edit directly. Thanks for calling it out.

The following characteristics apply:

* Server-side prerendering is supported.
* The `Value` parameter is only valid for the `Title` component.

> * All provided attributes are reflected in the rendered HTML tags.

Reviewer: This only applies to the `Meta` and `Link` components:

```suggestion
* All parameters provided to the `Meta` and `Link` components will be reflected in the rendered HTML tags.
```

> * For multiple `Title` components, the title of the page reflects the `Value` of the last `Title` component in the Razor markup.

Reviewer: This is true for most cases, but if any `Title` is re-rendered, it will change the page title regardless of its location in the markup (e.g. if a `Value` parameter changes, that change will be applied to the site).

```suggestion
* For multiple `Title` components, the title of the page reflects the `Value` of the last `Title` component to be rendered.
```

Maybe there's a better way to word this than what I've suggested?

> * If there are multiple `Meta` or `Link` components rendered with identical attributes, there is exactly one tag per component rendered. Two `Meta` or `Link` components can't refer to the same rendered HTML tag.

Reviewer: This property also applies when `Meta` or `Link` components _aren't_ rendered with identical attributes.

```suggestion
* Even if there are multiple `Meta` or `Link` components rendered with identical attributes, there is exactly one tag per component rendered. Two `Meta` or `Link` components can't refer to the same rendered HTML tag.
```
